driver/vhci/usb.ids

```diff
@@ -0,0 +1,19642 @@
+#
```

Comment:
`drivers/vhci/usb.ids` is required? We already have usb.ids at `userspace/usb.ids`.

Reply:
No is not. It's a mistake. Can be removed.